### PR TITLE
Update contributors.js

### PR DESCRIPTION
**Pull request by bottercode**

### CODE REVIEW

File: src/contributors.js [hunk 1]
```diff
@@ -185,3 +185,7 @@ export const adilRafeek = {
   username: "adilrafeek"
 };
 
+export const PSathyaNarayan = {
```
Comment (AbhayVAshokan) [inline src/contributors.js:188]: Please change the variable name to yours.

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]

**Review by bottercode — COMMENTED** [review]
done

File: src/contributors.js [hunk 2]
```diff
@@ -185,3 +185,7 @@ export const adilRafeek = {
   username: "adilrafeek"
 };
 
+export const DevanshDwivedi = {
+  name: "Devansh Dwivedi",
+  username: "port ghost"
```
Comment (AbhayVAshokan) [inline src/contributors.js:190]: One more change. Sorry I missed this earlier.  Change it to your GitHub id "bottercode". Click the preview link posted by Netlify above. It currently shows undefined below your name.

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]

**Review by bottercode — COMMENTED** [review]
done

**Pull request merged**